automated_testing/Makefile

```diff
@@ -0,0 +1,11 @@
+CHANNEL ?= stable
```

Comment:
I think this `automated_testing` directory would be better located in a folder under `test/end-to-end/` maybe called `multi-supervisor`?